apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/bp/BarnepensjonInnvilgelse.kt

```diff
@@ -53,6 +54,7 @@ data class BarnepensjonInnvilgelse(
                 beregning =
                     barnepensjonBeregning(innhold, avdoede, utbetalingsinfo, grunnbeloep, beregningsperioder, trygdetid),
                 etterbetaling = etterbetaling?.let { dto -> Etterbetaling.fraBarnepensjonDTO(dto) },
+                frivilligSkattetrekk = brevutfall.frivilligSkattetrekk ?: false,
```

Comment:
Man kunne evt. gjort som dette her da den alltid skal finnes på barnepensjon
```
frivilligSkattetrekk = brevutfall.frivilligSkattetrekk!!,
```

Reply:
Tenkjer vi berre kan gjera det sånn med ein gong, det kjem sikkert til å skje uforventa scenarie her også: `requireNotNull(brevutfall.frivilligSkattetrekk) { "Behandling ${brevutfall.behandlingId} mangler informasjon om frivillig skattetrekk, som er påkrevd for barnepensjon" }`

Reply:
@madsop-nav : Man får vel ikke så veldig god feilmelding ut hvis man bruker en av de definerte exception-typene? Feks `InternfeilException`?

Reply:
Sant nok, no tenkte eg mest for vår del i loggane. Men det er eit godt poeng at det jo bør vera synleg og forståeleg til saksbehandlar.

Reply:
@madsop-nav og @henrikg, er det bedre å ta i bruk `InternfeilException` slik jeg nå har gjort enn det som første ble foreslått?

Reply:
Eg tenkjer det viktige her er at det i så fall trynar med ein exception heller enn at vi svelgjer unna feilen, så og at dette er fint. Dette skal jo ikkje skje, og ikkje komme fram til saksbehandlar, med mindre noko skurrar skikkeleg.